recommended.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+const prettier = require('./eslint-plugin-prettier');
+const prettierConfig = require('eslint-config-prettier');
+
+module.exports = {
```

Comment:
https://github.com/prettier/eslint-plugin-prettier/blob/17c3c109cb9022301e804e418cae38ff489f8e69/eslint-plugin-prettier.js#L79


Please use `// @ts-check` and jsdoc for type checking to align with `.d.ts`

Reply:
I'll give it a try!